lib/router/config-router.js

```diff
@@ -110,7 +110,10 @@ function ConfigRouter(conf, schedulerRegistry) {
     this.disabledProtocols.push('websocket');
   }
 
-  this.reconnect = !conf.getValues('disable_reconnect').enabled;
+  this.reconnect = false;
```

Comment:
Is there a way to specify the default value of a configuration property? I see:
> `  this.getValue = function(criteria, defaultValue, inherit) {
It certainly would make this type of thing easier.